doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3381,27 +3381,6 @@ the conversion in hypotheses :n:`{+ @ident}`.
    never unfolded by tactics like :tacn:`simpl` and :tacn:`cbn` or to
    ensure that unfolding does not fail.
 
-   .. note::
-
-      This tactic unfortunately does not yet play well with tactic
-      internalization, resulting in interpretation-time errors when
-      you try to use it directly with opaque identifiers, as seen in
-      the first (failing) use of :tacn:`with_strategy` in the
-      following example.  This can be worked around by binding the
-      identifier to an |Ltac| variable, and this issue should
-      disappear in a future version of |Coq|; see `#12179
-      <https://github.com/coq/coq/issues/12179>`_.
-
-   .. example::
-
-      .. coqtop:: all reset abort
-
-         Opaque id.
-         Goal id 10 = 10.
-         Fail unfold id.
-         Fail with_strategy transparent [id] unfold id.
```

Comment:
Can we leave this example in as an example of the tactic, and simply remove the textual note, the Fail on this line, and the workaround on the next line?